.github/workflows/sycl_linux_build_and_test.yml

```
@@ -110,6 +110,18 @@ jobs:
       run: |
         export XDG_CACHE_HOME=$GITHUB_WORKSPACE/os_cache
         cmake --build $GITHUB_WORKSPACE/build --target check-clang
+    - name: check-mlir-sycl
+      if: ${{ always() && !cancelled() && steps.build.outcome == 'success' }}
+      run: |
+        cmake --build $GITHUB_WORKSPACE/build --target check-mlir-sycl
```

```suggestion
    - name: check-mlir
      if: ${{ always() && !cancelled() && steps.build.outcome == 'success' }}
      run: |
        cmake --build $GITHUB_WORKSPACE/build --target check-mlir
    - name: check-mlir-sycl
      if: ${{ always() && !cancelled() && steps.build.outcome == 'success' }}
      run: |
        cmake --build $GITHUB_WORKSPACE/build --target check-mlir-sycl
```

not directly needed but depends on it, so might be better to have it checked

---

Agree. I will add a step to check that mlir tests are clean.